pyntcloud/structures/delaunay.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, points,
                  furthest_site=False,
                  incremental=False,
                  qhull_options=None):
-        Structure.__init__(self, points)
```

Comment:
Nice catch :)

Reply:
Aah, yes, this one was hidden by the misspelled `delaunay.py`.